examples_esp/getting_started/platformio.ini

```diff
@@ -7,6 +7,11 @@
 ;
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+extra_configs =
+  platformio_override.ini
+
```

Comment:
here again. This file doesn't exist in RF24 repo.

Reply:
That definition is there so that someone can add configuration for their custom board, for example, without it being tracked by Git. Everything should work fine even when this file does not exist, no?

In my case I use the `platform_override.ini` to specify a custom path for the `.platformio` framework folder.

Reply:
ok, yeah it doesn't hurt as long as PIO doesn't output any warnings or errors.